apps/test-editor/.env

```
@@ -0,0 +1,6 @@
+VITE_OIDC_AUTHORITY=http://authentik.notopia.localhost/application/o/app-test-editor
+VITE_OIDC_CLIENT_ID=app-test-editor
+VITE_OIDC_REDIRECT_URI=http://localhost:4201
+VITE_OIDC_SCOPE=openid profile email
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)